test/WebSites/InlineConstraintsWebSite/Constraints/IsbnDigitScheme13Constraint.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Routing;
+
+namespace InlineConstraintsWebSite.Constraints
+{
+    public class IsbnDigitScheme13Constraint : IRouteConstraint
+    {
+        public bool Match(
+            HttpContext httpContext,
+            IRouter route,
+            string routeKey,
+            IDictionary<string, object> values,
+            RouteDirection routeDirection)
+        {
+            object value;
+
+            if (!values.TryGetValue(routeKey, out value))
+            {
+                return false;
+            }
+
+            var isbnNumber = value as string;
+
+            Func<char, bool> isValidIsbnChar = (char c) =>
```

Comment:
With chars I think you can just say `c >= '0' && c <= '9'` no? Also don't need to declare a lambda for this - just stick it inline.